addons/open-in-gandi/userscript.js

```diff
@@ -0,0 +1,12 @@
+export default async function ({ addon, console, msg }) {
+    const button = document.createElement("button");
+    button.className = "button sa-gd-button";
+    button.title = msg("open");
+
+    button.onclick = async (e) => {
+        const projectId = window.location.pathname.split("/")[2];
+        window.location.href = `https://scratch.mit.edu.host/projects/${projectId}/`;
```

Comment:
Why use a redirect link when you can just go straight there?

Reply:
i am not sure what you mean by "just go straight there" 😅

Reply:
i am probably wasting a lot of your time here...